src/EzPlatformEncoreBundle/bundle/Command/CompileAssetsCommand.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace EzSystems\EzPlatformEncoreBundle\Command;
+
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Process\Process;
+
+class CompileAssetsCommand extends ContainerAwareCommand
+{
+    protected function configure(): void
+    {
+        $this
+            ->setName('ezplatform:encore:compile')
```

Comment:
Can we use here  `public const EZPLATFORM_ENCORE_COMPILE_CMD_NAME = 'ezplatform:encore:compile';` ?

Reply:
Changed in d8f7857